vars/main.yml

```
@@ -423,6 +434,15 @@ wal_g_json:  # config https://github.com/wal-g/wal-g#configuration
 wal_g_archive_command: "wal-g wal-push %p"
 wal_g_patroni_cluster_bootstrap_command: "wal-g backup-fetch {{ postgresql_data_dir }} LATEST"
 
+# Define job_parts outside of wal_g_cron_jobs
+# ⚠️ Ensure there is a space at the beginning of each part to prevent commands from concatenating.
+wal_g_cron_create_job_parts:
```

```suggestion
wal_g_backup_command:
```

@ThomasSanson what do you say about such a variable name?

---

Indeed, I concur with your approach to variable naming

You possess a superior understanding of their scope compared to myself 😇

---

Done here : https://github.com/vitabaks/postgresql_cluster/pull/417/commits/b207edc47fb252933c7d10d99608d334b8c504d5